docs/framework/winforms/advanced/walkthrough-creating-an-accessible-windows-based-application.md

```diff
@@ -131,8 +131,10 @@ Creating an accessible application has important business implications. Many gov
     |MenuItem|Name|exitApp|  
     ||Text|E&xit|  
   
-     ![Pizza Order Form](./media/vbpizzaorderform.gif "vbPizzaOrderForm")  
-Your form will look something like the following:  
+  Your form will look something like the following image:  
```

Comment:
This needs to be indented otherwise the screenshot below renders as a code block